components/google_sheets/google_sheets.app.mjs

```diff
@@ -45,7 +49,7 @@ export default {
     sheetID: {
       type: "string",
       label: "Spreadsheet",
-      description: "The Google spreadsheet",
+      description: "The Spreadsheet ID",
```

Comment:
I think this prop's [description](https://pipedream.com/docs/components/guidelines/#descriptions) is intended to be removed (in https://github.com/PipedreamHQ/pipedream/pull/2019/commits/1609a5346950fba72c9f2c5a3ccf602e5e74fda4) because its label is self explanatory.

```suggestion
```

Reply:
I forgot to mention in my previous comment that I'm also getting the same ESLint error for the `sheetName` and `worksheetIDs` props.